code/modules/mob/living/carbon/human/species/station/traits_vr/neutral.dm

```diff
@@ -113,3 +113,12 @@
 	..(S,H)
 	H.verbs |= /mob/living/proc/glow_toggle
 	H.verbs |= /mob/living/proc/glow_color
+
+/datum/trait/antiseptic_saliva
+	name = "Antiseptic Saliva"
+	desc = "Your saliva has especially strong antiseptic properties that can be used to heal small wounds."
+	cost = 0
```

Comment:
Probably should make it cost at least 1.

Reply:
The logic for this is given in the PR itself--it doesn't make sense to me to charge for this when heal bellies are free and better.